update_accesslogs_option (#357)
CleverCloud/documentation: content/doc/administrate/log-management.md

clever accesslogs
```

As with the `logs` command, you can specify `--before` and `--after` flags.

> If no options are specified, the logs display continuously.  

Reviewer: ```suggestion
If you don't specify any options, the logs display continuously.  
```

> If no options are specified, the logs will be displayed continuously.  

Reviewer: ```suggestion
If no options are specified, the logs display continuously.  
```

> If you need to change the output you can specify the `--format` flag with one of these values:

Reviewer: ```suggestion
To change the output, specify the `--format` flag with one of these values:
```

clever drain remove [--alias <alias>] <DRAIN-ID>
```

> If the status of your drain is shown as `DISABLED` without you disabling it, it may be because we have not been able to send your logs to your drain endpoint or because the requests timed out after **25 seconds**.

Reviewer: ```suggestion
If the status of your drain appears as `DISABLED` without you disabling it, it may be because we have not been able to send your logs to your drain endpoint or because the requests timed out after **25 seconds**.
```

> You can also use the logs drain to send your add-on's logs by using `--addon` flag, the value must be the add-on id starting by `addon_`.

Reviewer: ```suggestion
Use the logs drain to send your add-on's logs by using `--addon` flag, the value must be the add-on id starting by `addon_`.
```
